warehouse/models/mart/ntd_fct_annual/_mart_ntd_fct_annual.yml

```diff
@@ -0,0 +1,102 @@
+version: 2
+
+x-common-fields:
```

Comment:
These common fields are not in use here. I think would be better to add when adding documentation.